app/config.js

```diff
@@ -0,0 +1,16 @@
+// Use this file to change prototype configuration.
+
+// Note: prototype config can be overridden using environment variables (eg on heroku)
+
+module.exports = {
+
+  // Service name used in header. Eg: 'Renew your passport'
+  serviceName: 'Set service name in app/config.js',
+
+  // Default port that prototype runs on
+  port: '3000',
+
+  // Enable or disable password protection on production
+  useAuth: 'true'
```

Comment:
@teneightfive I think we still need to do the lowercase conversion as we're also reading the var out of environment variables. In which case I'm unsure if it matters here?